### ISSUE TITLE
[Bug]: pod system-upgrade   pod/system-upgrade-controller- CrashLoopBackOff   

### ISSUE THREAD

**Issue by remotejob** [kube-hetzner/terraform-hcloud-kube-hetzner#1251]
### Description

Looks like   system-upgrade-controller don't start anymore
I tried 

`automatically_upgrade_os = false`

`automatically_upgrade_k3s = false`

But it won't help

image "rancher/system-upgrade-controller:v0.11.0"

log: 
Error starting: namespaces \"kube-system\" is forbidden: User \"system:serviceaccount:system-upgrade:system-upgrade\" cannot get resource \"namespaces\" in API group \"\" in the namespace \"kube-system\""

  

### Kube.tf file

```terraform
"apiVersion": "kustomize.config.k8s.io/v1beta1"
"kind": "Kustomization"
"patches":
- "patch": |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      name: system-upgrade-controller
      namespace: system-upgrade
    spec:
      template:
        spec:
          containers:
            - name: system-upgrade-controller
              volumeMounts:
                - name: ca-certificates
                  mountPath: /var/lib/ca-certificates
          volumes:
            - name: ca-certificates
              hostPath:
                path: /var/lib/ca-certificates
                type: Directory
  "target":
    "group": "apps"
    "kind": "Deployment"
    "name": "system-upgrade-controller"
    "namespace": "system-upgrade"
    "version": "v1"
- "path": "kured.yaml"
- "path": "ccm.yaml"
"resources":
- "https://github.com/hetznercloud/hcloud-cloud-controller-manager/releases/download/v1.19.0/ccm-networks.yaml"
- "https://github.com/kubereboot/kured/releases/download/1.15.0/kured-1.15.0-dockerhub.yaml"
- "https://raw.githubusercontent.com/rancher/system-upgrade-controller/master/manifests/system-upgrade-controller.yaml"
- "cert_manager.yaml"
```

### Platform

Linux

**Comment by sebastianlutter** [comment]
The issue is caused by changes to that have been committed yesterday https://raw.githubusercontent.com/rancher/system-upgrade-controller/master/manifests/system-upgrade-controller.yaml

This is discussed in https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/issues/1252

This pull request exists to fix this issue: https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/pull/1253

**Comment by mysticaltech** [comment]
@remotejob  Fix by @sebastianlutter was just released in v2.13.1  🚀
